assets/src/edit-story/app/prepublish/text.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import stripHTML from '../../utils/stripHTML';
+
+const maxPageCharacterCount = 200;
+const minStoryCharacterCount = 100;
+
+const characterCountForPage = (page) => {
+  let characterCount = 0;
+  page.elements.forEach((element) => {
+    if (element.type === 'text') {
+      characterCount += stripHTML(element.content).length;
+    }
+  });
+  return characterCount;
+};
+
+export const pageTooMuchText = (page) => {
+  const characterCount = characterCountForPage(page);
+
+  if (characterCount > maxPageCharacterCount) {
+    return {
+      message: 'Too much text on page',
```

Comment:
I forgot we should translate the text :) see [here](https://github.com/google/web-stories-wp/pull/5029/files#diff-5ea3e91693715472e1edeff0dafe175ca48e6b0f0b72153d8a89a189a4cd4133R20)

Reply:
Thanks! I was going to bring that up today! Is there a doc with finalized text for these guidelines?

Reply:
I don't think so. All I've seen is the [product brief](https://docs.google.com/document/d/1GJZljicdXcP7oRdywsF3EGUaCVpCHsDrK2gUMubnrbk/edit?pli=1#) and the tickets themselves